tests/_test_base.bash

```diff
@@ -66,8 +72,8 @@ function test_user_password {
 
 function stop_gpg_agent {
   local username=$(id -u -n)
-  ps -wx -U "$username" | gawk \
-    '/gpg-agent --homedir/ { if ( $0 !~ "awk" ) { system("kill -9 "$1) } }' \
+  ${PS_CMD} "$username" | gawk \
+    '/gpg-agent --homedir/ { if ( $0 !~ "awk" ) { system("echo -9 "$1) } }' \
```

Comment:
why this change from 'kill' to 'echo' ?

Reply:
Sorry, some lingering debug code. Fixed below